code/game/data_huds.dm

```diff
@@ -314,3 +317,87 @@
 			holder.icon_state = "hudmove"
 		else
 			holder.icon_state = ""
+
+/*~~~~~~~~~~~~~~
+	PLANT HUD
+~~~~~~~~~~~~~~~*/
+/proc/RoundPlantBar(value)
+	switch(value * 100)
+		if(91 to INFINITY)
+			return "max"
+		if(81 to 90)
```

Comment:
This switch has gaps. If you pass in a `value` of 0.905, it'll become 90.5 and fall straight through to the else. Compare to the other `RoundWhatever` procs in this file, which don't have gaps.